### ISSUE TITLE
Limit the number of users per room

### ISSUE THREAD

**Issue by evilword666** [lynckia/licode#976]
I want to make that in each room that I create only two users can be added as maximum.

I have in my client this code:

     RoomX.addEventListener ('room-connected', function (roomEvent) {
       Var options = {metadata: {type: 'publisher'}};
       Var enableSimulcast = getParameterByName ('simulcast');
       If (enableSimulcast) options.simulcast = {numSpatialLayers: 2};

But I think that is not where I should modify the parameter of the maximum number of users per room because I can add more.

Could anyone help me?

**Comment by arsenicraghav** [comment]
@evilword666 
Yes you are right, you should not include that restriction logic on the client side. You should do a function on the server side which would check the number of users for the given room and return true or false accordingly. or you may want to have a look at the roomController.js where you can probably limit the subscribers and other stuff.

**Comment by jcague** [comment]
there isn't a way to limit the number of users in a room currently. But you can always do that in your app.

**Comment by sebastianfelipe** [comment]
Can I leave the limit_n_rooms infinite or something? In that case, what is the value I need to set?

Thanks.
